File: 191. Number of 1 Bits.py
class Solution:
    def hammingWeight(self, n: int) -> int:
        count = 0

        while n > 0:
            if n & 1 == 1:
                count += 1
            n = n >> 1

        return count


newHamming = Solution()
print(newHamming.hammingWeight(11))

File: 191_Number_of_1_Bits.c
#include <stdio.h>

int hammingWeight(int n)
{
    int count = 0;
    while (n != 0)
    {
        if (n & 1 == 1)
            count++;
        n = n >> 1;
    }
    return count;
}

int main()
{
    int oneBit = hammingWeight(11);
    printf("%d", oneBit);

    return 0;
}